mlflow/fastai.py

```diff
@@ -157,7 +158,10 @@ def __init__(self, learner):
     def predict(self, dataframe):
         test_data = TabularList.from_df(dataframe, cont_names=self.learner.data.cont_names)
         self.learner.data.add_test(test_data)
-        return self.learner.get_preds(DatasetType.Test)[0].numpy()
+        preds, target = self.learner.get_preds(DatasetType.Test)
+        preds = pd.Series(map(tuple, preds.numpy()), name='predictions')
```

Comment:
```suggestion
        preds = pd.Series(map(np.array, preds.numpy()), name='predictions')
```

This will make the resulting DataFrame convertible into a Spark DataFrame when used with the Spark UDF functionality,